community/cypher/cypher-compiler-3.0/src/main/scala/org/neo4j/cypher/internal/compiler/v3_0/pipes/SortPipe.scala

```diff
@@ -61,3 +62,28 @@ case class SortPipe(source: Pipe, orderBy: Seq[SortDescription])
 
   def withEstimatedCardinality(estimated: Double) = copy()(Some(estimated))
 }
+
+private object emptyOrdering extends scala.Ordering[ExecutionContext] {
+  override def compare(x: ExecutionContext, y: ExecutionContext): Int = -1
+}
+
+private class InnerOrdering(order: Seq[SortDescription])(implicit qtx: QueryState) extends scala.Ordering[ExecutionContext] {
```

Comment:
No need for `implicit` here either. In fact, no need for the `QueryState` at all, it looks like.